snmp/datadog_checks/snmp/data/default_profiles/checkpoint-watchguard.yaml

```diff
@@ -0,0 +1,9 @@
+extends:
+  - checkpoint.yaml
+  - _generic-host-resources-base.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.3097.1.5.*
+  - 1.3.6.1.4.1.3097.1.5.87
+  - 1.3.6.1.4.1.3097.1.5.96
+  - 1.3.6.1.4.1.3097.1.5.94
+  - 1.3.6.1.4.1.3097.1.5.93
```

Comment:
why specify this when it will be already covered with `1.3.6.1.4.1.3097.1.5.*` ?

Reply:
@NouhaManai96 It's not indeed necessary indeed, but doesn't hurt either.